### PR TITLE
Mention Görli in the docs

### PR DESCRIPTION
**Pull request by ncocchiaro**
Added Görli to supported networks, merged two questions in the WS FAQ since they may become ambiguous given the different spellings of network names (görli/goerli), and moved Kovan down in "Choose a network" to be consistent with the ordering in other parts of the site.

### CODE REVIEW

File: docs/ethereum/wss/faq.md [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
+There is an endpoint for each supported network (Mainnet, Ropsten, Rinkeby, Kovan, Görli):
 
-**What are the supported networks?**
-
-Mainnet, Ropsten, Rinkeby, Kovan
+- `wss://mainnet.infura.io/ws`
+- `wss://ropsten.infura.io/ws`
+- `wss://rinkeby.infura.io/ws`
+- `wss://kovan.infura.io/ws`
+- `wss://goerli.infura.io/ws`
```
Comment (JeeChoi) [inline docs/ethereum/wss/faq.md]: should we add /v3/<project_ID> to the end of these endpoints?

**Comment by ryanschneider** [comment]
GH is acting up for me and won't let me reply to Jee's suggestion RE: v3 ids, but my suggestion would be to just rephrase the FAQ answer like so:

> There is an endpoint for each supported network (Mainnet, Ropsten, Rinkeby, Kovan, Görli), see the [Getting Started] guide for endpoints.

^ Preview is also broken so can't see how that formatted either :/

**Comment by ncocchiaro** [comment]
Makes sense, updated with Ryan's suggestion so we also cover Jee's suggestion with a single source of truth.

**Comment by ryanschneider** [comment]
lgtm

**Pull request merged**